tests/annotator/annotator.cpp

```diff
@@ -1911,42 +1984,21 @@ TEST(Annotator, hashChangesAndUpdates)
     auto model = parser->parseModel(modelStringUniqueIds);
     auto annotator = libcellml::Annotator::create();
 
-    EXPECT_FALSE(annotator->isBuilt());
-
-    annotator->buildModelIndex(model);
-    EXPECT_TRUE(annotator->isBuilt());
+    EXPECT_FALSE(annotator->hasModel());
 
-    model->setId("iHaveChangedSinceTheLastBuild");
-    EXPECT_FALSE(annotator->isBuilt());
+    // Model is set and the id map is not created.
```

Comment:
The id map is **not** created?? I thought it was built when the model was set??

Reply:
Is this comment still relevant? @kerimoyle? @hsorby?